util/generate-seed.js

```diff
@@ -3,17 +3,46 @@
  */
 'use strict';
 
-const {decode} = require('base58-universal');
+const {encode, decode} = require('base58-universal');
 const {driver} = require('@digitalbazaar/did-method-key');
 const {generateId} = require('bnid');
 
+// multibase base58-btc header
+const MULTIBASE_BASE58BTC_HEADER = 'z';
+// multihash identity function cdoe
+const MULTIHASH_IDENTITY_FUNCTION_CODE = 0x00;
+// seed byte size
+const SEED_BYTE_SIZE = 32;
+const SEED_BITS_SIZE = SEED_BYTE_SIZE * 8;
+
 const didKeyDriver = driver();
 
 (async () => {
-  const seedBase58 = await generateId({bitLength: 256, multibase: false});
-  console.log(`SECRET seedBase58: ${seedBase58}`);
+  // generate a random base58 encoded 256 Bit (32 Byte) seed
+  const seedBase58 = await generateId({
+    bitLength: SEED_BITS_SIZE, multibase: false
+  });
   // convert base58 to Uint8Array
-  const seed = decode(seedBase58);
-  const didKey = await didKeyDriver.generate({seed});
+  const seedBytes = decode(seedBase58);
+  if(seedBytes.length !== SEED_BYTE_SIZE) {
```

Comment:
Why is this check necessary? We trust `generateId` or we don't, right?

Reply:
Can't we just use bnid's IdGenerator and avoid having to decode here?

Reply:
We can use it. Pushing an update.